### PR TITLE
Add stamps api

### PR DESCRIPTION
**Pull request by sayan-gg**
Changes summary:
(1) Added an endpoint to get all stamps
(2) Fixed a tab issue now that we've agreed upon 4-space tabs
(3) Moved the `.vscode/` directory into `.gitignore`

Open Questions
(1) Do we want to have the `.vscode/` directory (which includes `settings.json` in the `.gitignore` file or do we want to include it so that the configs are included in the repo? 

Testing I Did Before Pushing:
(1) Tested all URLs (populate datatables, fetch hike, fetch list of stamps) to make sure they were all working before I submitted this PR. I think we should decide on a testing framework but for now, these are the tests I'm going to do to make sure I'm not breaking anything obvious

### CODE REVIEW

File: gorapass/views.py [hunk 1]
```diff
@@ -12,6 +12,10 @@
 def index(request):
     return HttpResponse('Hello, World. This is Naya and Brandi\'s super cool app.')
 
+def stamps(request):
+    stamp_model = list(Stamps.objects.values())
```
Comment (brandiseeley) [inline gorapass/views.py:16]: This looks good. The only thing that comes to mind is what happens if the stamps list is empty. With this code, we will be given an empty list, where we might want to return a 404 instead. Not likely to happen, but I think it's a good habit. `django.shortcuts` provides an equivalent `get_list_or_404` that does the same as `get_object_or_404`, but for collections.

What do you think?

**Review by brandiseeley — COMMENTED** [review]
Looks good, I just have one comment about the stamps endpoint.

File: gorapass/views.py (hunk 1 above)
Comment (brandiseeley) [reply gorapass/views.py:16]: Thinking on this some more, I think that returning an empty list makes sense. We 404 when we can't find something specific. We can find the hikes, there just aren't any. I think this is good.

**Review by brandiseeley — APPROVED** [review]
👍

File: gorapass/views.py (hunk 1 above)
Comment (sayan-gg) [reply gorapass/views.py:16]: I agree. I think we should return an empty list. We are aligned 👍

**Pull request merged**